.github/workflows/test.yml

```diff
@@ -153,6 +153,24 @@ jobs:
           name: coverage-data-${{ github.job }}-${{ strategy.job-index }}
           path: .coverage.*
           include-hidden-files: true
+      - name: Test project template
+        if: ${{ matrix.experimental }}
+        env:
+          PYTHONWARNINGS: error
+        run: |
+          wagtail start testproject
+          cd testproject
+          # Remove when all experimental jobs use Django >= 6.0
+          tee -a testproject/settings/local.py << EOF
+          from warnings import filterwarnings
+          filterwarnings(
+              "ignore", "The FORMS_URLFIELD_ASSUME_HTTPS transitional setting is deprecated."
+          )
+          FORMS_URLFIELD_ASSUME_HTTPS = True
+          EOF
```

Comment:
This is awkward, but necessary for now. Adam Johnson wrote an explanation in his blog post: https://adamj.eu/tech/2023/12/07/django-fix-urlfield-assume-scheme-warnings